src/modules/ie-objects/hooks/get-ie-objects-info.ts

```diff
@@ -8,13 +9,20 @@ import { IeObject } from './../types';
 export const useGetIeObjectsInfo = (
 	id: string,
 	options: {
-		keepPreviousData: boolean;
-		enabled: boolean;
-	} = { keepPreviousData: true, enabled: true }
-): UseQueryResult<IeObject> => {
+		keepPreviousData?: boolean;
+		enabled?: boolean;
+	} = {}
+): UseQueryResult<IeObject | null> => {
 	return useQuery(
 		[QUERY_KEYS.getIeObjectsInfo, { id }],
-		() => IeObjectsService.getById(id),
-		options
+		() => {
+			if (isStringOfFormatIeObjectSchemaIdentifier(id)) {
+				// If the id doesn't have the correct format, we don't even have to try the request to the backend
+				return null;
+			}
+
+			return IeObjectsService.getById(id);
+		},
+		{ keepPreviousData: true, enabled: true, ...options }
```

Comment:
Geen optie om hier `enabled` te gebruiken ipv `null` return?
https://tanstack.com/query/v4/docs/framework/react/guides/disabling-queries

Reply:
dan blijft die op loading staan
maar ik wil dat die direct zegt: geen data, zodat we niet wachten om de 404 page te tonen